src/rest/server.rs

```diff
@@ -3,7 +3,8 @@ use crate::blockchain::transaction::{PubKey, Signature, Transaction};
 use crate::blockchain::util::Timestamp;
 use base64::{decode_config, encode_config};
 use futures::channel::oneshot;
-use rocket::{self, config, config::Config, http::Status, response::status, *};
+use rocket::{self, config, config::Config, http::Status, http::Method, get, routes, response::status, *};
+use rocket_cors::{self, AllowedHeaders, AllowedOrigins, Cors, CorsOptions};
```

Comment:
The "Cors" and "CorsOptions" are unused. The reason is that on line 23 you specify the namespace "rocket_cors", which is not necessary when importing the type.

```suggestion
use rocket_cors::{self, AllowedHeaders, AllowedOrigins};
```
or alternatively modify line 23